cmd/utils/git.go

```diff
@@ -27,7 +28,10 @@ import (
 	"k8s.io/utils/pointer"
 )
 
-var isOktetoSample *bool
+var (
+	isOktetoSample *bool
+	mu             sync.Mutex
```

Comment:
Can we rename it so we know it's only used by `IsOktetoRepo`? Maybe `IsOktetoRepoMutex` or `IsOktetoRepoLock`

Reply:
i moved to the function